modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your ServiceMeshControlPlane resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before you deploy your service mesh in production.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. For production deployments you _must_ customize the default template. 
```

Comment:
Comma after `deployments` So, `For production deployments, you must customize the default template.`